dynoid/dynoid.go

```diff
@@ -60,9 +60,9 @@ type IssuerCallback func(issuer string) error
 
 // AllowHerokuHost verifies that the issuer is from Heroku for the given host
 // domain
-func AllowHerokuHost(host string) IssuerCallback {
+func AllowHerokuHost(issuerHost string) IssuerCallback {
```

Comment:
I think this change is a bit misleading as it's not the issuer host but the `HEROKU_HOST` equivalent you're providing.